interface/patient_file/summary/demographics.php

```diff
@@ -1498,7 +1490,7 @@ function setMyPatient() {
          $widgetLabel = "recurrent_appointments";
          $widgetButtonLabel = xl("Add");
          $widgetButtonLink = "return newEvt();";
-         $widgetButtonClass = "";
+         $widgetButtonClass = "btn-add";
          $linkMethod = "javascript";
          $bodyClass = "summary_item small";
```

Comment:
check out the gui for several places where the Edit link is missing the pencil icon